client/src/hooks/helpers/useIsAllocatedTo.ts

```diff
@@ -0,0 +1,25 @@
+import { useMemo } from 'react';
+
+import { Response } from 'hooks/queries/useUserAllocations';
+
+export default function useIsAllocatedTo(
+  address: string,
+  allocations: string[],
+  epoch: number,
+  isDecisionWindowOpen: boolean,
+  userAllocations: Response | undefined,
+): boolean {
+  return useMemo(() => {
```

Comment:
`useMemo` is unnecessary

Reply:
The original idea for this `useMemo` came from you IIRC. The idea is that user can toggle heart button as much as they want w/o recalculation.